cookiecutter.json

```diff
@@ -14,5 +14,40 @@
         "GPLv3",
         "Apache Software License 2.0"
     ],
-    "use_bucketeer_aws_for_file_storage": "y"
+    "use_bucketeer_aws_for_file_storage": "y",
+    "node_version": [
+        "11.11.0",
+        "11.10.1",
+        "11.9.0",
+        "11.8.0",
+        "11.7.0",
+        "11.6.0",
+        "11.5.0",
+        "11.4.0",
+        "11.3.0",
+        "11.2.0",
+        "11.1.0",
+        "11.0.0",
+        "10.15.3",
+        "10.14.2",
+        "10.13.0",
+        "10.12.0",
+        "10.11.0",
+        "10.10.0",
+        "10.9.0",
+        "10.8.0",
+        "10.7.0",
+        "10.6.0",
+        "10.5.0",
+        "10.4.1",
+        "10.3.0",
+        "10.2.1",
+        "10.1.0",
+        "10.0.0"
+    ],
+    "yarn_version": [
+        "1.14.0",
+        "1.13.0",
+        "1.12.3"
+    ]
```

Comment:
Is this a list of options that the person running cookiecutter can select from? I think it would make more sense to specify a single known good version as a default, but allow entering a different version as text. That way we don't have to maintain this list every time there's a new node or yarn release.